src/main/groovy/edu/oregonstate/mist/trailsapi/resources/TrailResources.groovy

```diff
@@ -46,9 +46,19 @@ public class TrailsResource extends Resource {
 	    )
 	}
 
+	ResultObject trailResult(List<Trail> trails) {
+		List<ResourceObject> trailResourceObjects = []
```

Comment:
No need to change it if you don't want to, but this few lines could be shortened to something like:
```
List<ResourceObject> trailResourceObjects = trails.collect { trail -> trailResource(trail)) }
```
or even just
```
new ResultObject(data: trails.collect { trail -> trailResource(trail)) })
```